tests/install.sh

```diff
@@ -0,0 +1,15 @@
+set -ex
+
+CMAKE_PRESET=$1
+CMAKE_INSTALL_PREFIX=$2
+
+echo "> Running install test for CMake preset ${CMAKE_PRESET}. Installation is expected to happen in ${CMAKE_INSTALL_PREFIX}."
+
+cmake --build --preset=${CMAKE_PRESET} -j8 --target=install
```

Comment:
I wonder if we could do here 

```
cmake --install build-with-${CMAKE_PRESET} -j8 --prefix ${CMAKE_INSTALL_PREFIX}
```

to avoid that we have to add `-DCMAKE_INSTALL_PREFIX=/opt/test-folder-for-install` in the configure and build step in the GitHub workflow.

Reply:
:heavy_check_mark: